src/Psalm/Internal/Analyzer/Statements/Expression/Fetch/VariableFetchAnalyzer.php

```diff
@@ -38,7 +38,6 @@ class VariableFetchAnalyzer
         '$_SESSION',
         '$_REQUEST',
         '$_ENV',
-        '$http_response_header',
```

Comment:
This is used in other places as well (via `VariableFetchAnalyzer::isSuperGlobal()`) so it's safer to keep it on this list and just override the type, like you did below.